wire-ios-sync-engine/Source/E2EI/CRL/CertificateRevocationListsChecker.swift

```diff
@@ -112,6 +109,7 @@ public class CertificateRevocationListsChecker: CertificateRevocationListsChecki
 
     private func checkCertificateRevocationLists(from distributionPoints: Set<URL>) async {
 
+        var shouldNotifyAboutRevokedCertificate = false
```

Comment:
did you mean to rename this `checkE2EICertificateForSelfClient` ?

Reply:
Ah no I see it's the other way around... You applied @netbe's suggestion but it didn't apply to the other occurrences of `checkE2EICertificateForSelfClient` so it won't be able to build